code/modules/power/cable.dm

```diff
@@ -53,25 +53,25 @@ var/list/possible_cable_coil_colours
 	return PN.draw_power(amount)
 
 /obj/structure/cable/yellow
-	color = COLOR_YELLOW
+	color = COLOR_AMBER
```

Comment:
why `COLOR_AMBER` and not `COLOR_YELLOW`?

Reply:
Oh, I believe I ported the more dark colored cables by accident, but I think they look neater, and not as bright, have you ever seen neon cables?

Reply:
Totally unnecessary.

Reply:
It is. That's usual. Cables being bright and visible is always good.

Reply:
Will undo